pkg/skaffold/kubernetes/portforward/forwarder_manager.go

```diff
@@ -39,11 +41,12 @@ type Forwarder interface {
 
 // ForwarderManager manages all forwarders
 type ForwarderManager struct {
+	iteration  int
 	forwarders []Forwarder
 }
 
 // NewForwarderManager returns a new port manager which handles starting and stopping port forwarding
-func NewForwarderManager(out io.Writer, cli *kubectl.CLI, podSelector kubernetes.PodSelector, namespaces []string, label string, runMode config.RunMode, options config.PortForwardOptions, userDefined []*latest.PortForwardResource) *ForwarderManager {
+func NewForwarderManager(out io.Writer, cli *kubectl.CLI, podSelector kubernetes.PodSelector, iteration int, label string, runMode config.RunMode, options config.PortForwardOptions, userDefined []*latest.PortForwardResource) *ForwarderManager {
```

Comment:
I was under the impression, `NewForwarderManager`  was called only once at the start `devloop`. Can you confirm if you are seeing events for subsequent dev loops with correct dev iteration?

Reply:
You're right, I meant to come back to this PR and update it as this approach doesn't work. In https://github.com/GoogleContainerTools/skaffold/pull/5710 I wanted to add the iteration field to the event handler and have a function to grab that value so that it doesn't have to be plumbed down into unrelated skaffold logic like I'm doing here